pkg/process/events/model/model_linux.go

```diff
@@ -0,0 +1,48 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+//go:generate go run github.com/tinylib/msgp -tests=false
+
+package model
+
+import (
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/security/secl/model"
+)
+
+// ProcessMonitoringEvent is an event sent by the ProcessMonitoring handler in the runtime-security module
+type ProcessMonitoringEvent struct {
+	*model.ProcessCacheEntry
+	EventType string    `json:"EventType" msg:"evt_type"`
+	Date      time.Time `json:"Date" msg:"date"`
+}
+
+// ProcessMonitoringtoProcessEvent converts a ProcessMonitoringEvent to a generic ProcessEvent
+func ProcessMonitoringtoProcessEvent(e *ProcessMonitoringEvent) *ProcessEvent {
+	var cmdline []string
+	if e.ArgsEntry != nil {
+		cmdline = e.ArgsEntry.Values
+	}
+
+	return &ProcessEvent{
+		EventType: e.EventType,
+		Date:      e.Date,
```

Comment:
Just wondering is this `Date` the event time?

Reply:
Yes, it's the date when the event has been collected by eBPF.

Reply:
I'm going to rename this field to `CollectionTime` to make it a bit more explicit.

Reply:
https://github.com/DataDog/datadog-agent/pull/11782/commits/2249dadfb6475d5e9c095ad872e8ba40aded9f61